x/poolmanager/router_test.go

```diff
@@ -1266,6 +1326,48 @@ func (suite *KeeperTestSuite) calcInAmountAsSeparateSwaps(osmoFeeReduced bool, r
 	}
 }
 
+// calcInAmountAsSeparateBalancerSwaps calculates the output amount of a series of swaps on Concentrated pools while factoring in reduces swap fee changes.
+// It uses CL functions directly to ensure the poolmanager functions route to the correct modules.
+func (suite *KeeperTestSuite) calcInAmountAsSeparateConcentratedSwaps(osmoFeeReduced bool, routes []types.SwapAmountInRoute, tokenIn sdk.Coin) sdk.Coin {
```

Comment:
We should be able to combine this function with `calcInAmountAsSeparateBalancerSwaps` instead of duplicating by calling the relevant methods on the poolmanager keeper instead of gamm or concentrated liquidity keepers

Reply:
fixed